src/components/buttonPanel.jsx

```diff
@@ -1,5 +1,6 @@
 import React, { Component } from 'react';
 import Button from './button';
+import '../css/buttonPanel.css';
 
 class ButtonPanel extends Component {
```

Comment:
`ButtonPanel` seems to be a stateless component, is there a reason we using class component? :thinking:

Reply:
I have modified that already in Milestone 4. Thanks for the pointer :thumbsup: